website/docs/d/azure_access_credentials.html.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: "vault"
+page_title: "Vault: vault_azure_access_credentials data source"
+sidebar_current: "docs-vault-datasource-azure-access-credentials"
+description: |-
+  Reads Azure credentials from an Azure secret backend in Vault
+---
+
+# vault\_azure\_access\_credentials
+
+Reads Azure credentials from an Azure secret backend in Vault.
+
+~> **Important** All data retrieved from Vault will be
+written in cleartext to state file generated by Terraform, will appear in
+the console output when Terraform runs, and may be included in plan files
+if secrets are interpolated into any resource attributes.
+Protect these artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+## Description
+
+The Azure Active Directory data source exists to easily pull short-lived
+credentials from Vault for use in Terraform. By default, it returns a 
+dynamically generated `client_id` and `client_secret` without testing 
+whether they've fully propagated for use in Azure Active Directory. However,
+by activating `validate_creds`, credentials will be tested before being 
+returned. This will, however, increase the time it takes for the credentials
+to be returned, blocking Terraform's execution until they are ready.
+
+If `validate_creds` is used, by default, credentials will be validated by 
+making a test call to Azure every 7 seconds. When we have received 8 
+successes in a row, the credentials will be returned. We have seen propagation 
+times take up to 15 minutes, so the maximum length of time for the check defaults 
+to 20 minutes.
+
+However, propagation times will vary widely based on each company's Azure
+usage, so all these settings are configurable.
+
+## Example Usage
+
+```hcl
+data "vault_azure_access_credentials" "creds" {
+  role    = "my-role"
+  validate_creds = true
+  num_sequential_successes = 8
+  num_seconds_between_tests = 7
+  max_cred_validation_seconds = 1200 // 20 minutes
+  subscription_id = "291bba3f-e0a5-47bc-a099-3bdcbexample"
+  tenant_id = "291bba3f-e0a5-47bc-a099-3bdcbexample"
+}
+
+provider "azure" {
+  client_id = "${data.vault_azure_access_credentials.creds.client_id}"
+  client_secret = "${data.vault_azure_access_credentials.creds.client_secret}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `backend` - (Required) The path to the Azure secret backend to
+read credentials from, with no leading or trailing `/`s.
+
+* `role` - (Required) The name of the Azure secret backend role to read
+credentials from, with no leading or trailing `/`s.
+
+* `validate_creds` - (Optional) Whether generated credentials should be 
+validated before being returned. Defaults to `false`, which returns 
+credentials without checking whether they have fully propagated throughout
+Azure Active Directory. Designating `true` activates testing. Defaults
```

Comment:
"Default to `false`" has already been mentioned.

Reply:
:+1: https://github.com/terraform-providers/terraform-provider-vault/pull/713/commits/3d64cb5b713f456488b2799ed4cd1d1892ef258f